src/CattyTests/DataModel/UserVariableTests.swift

```diff
@@ -70,4 +73,91 @@ final class UserVariableTests: XCTestCase {
         XCTAssertFalse(userVariableA === userVariableCopy)
         XCTAssertTrue(userVariableB === userVariableCopy)
     }
+
+    func testIsEqualToUserVariableForDirectAssignment() {
+        let userVariableA = UserVariable()
+        userVariableA.name = "userVar"
+        userVariableA.value = self.arrayWithSameElement
+
+        var userVariableB = userVariableA
```

Comment:
Please do not do that, this makes the test useless. Init a new instance for `userVariableB`.